lib/scanner/walk.go

```diff
@@ -101,10 +100,6 @@ type walker struct {
 func (w *walker) walk(ctx context.Context) (chan protocol.FileInfo, error) {
```

Comment:
This (and `Walk`) can now drop the `error` return value (all returns just pass nil).